src/drc/api/tests/test_audittrails.py

```diff
@@ -0,0 +1,199 @@
+import uuid
+from base64 import b64encode
+from unittest.mock import patch
+from freezegun import freeze_time
+from datetime import datetime
+
+from django.test import override_settings
+from django.urls import reverse, reverse_lazy
+
+from rest_framework.test import APITestCase
+from vng_api_common.audittrails.models import AuditTrail
+from vng_api_common.constants import ObjectTypes
+
+from drc.datamodel.models import EnkelvoudigInformatieObject
+from drc.datamodel.tests.factories import EnkelvoudigInformatieObjectFactory
+
+ZAAK = f'http://example.com/zrc/api/v1/zaken/{uuid.uuid4().hex}'
+
+@freeze_time('2019-01-01')
+@override_settings(LINK_FETCHER='vng_api_common.mocks.link_fetcher_200')
+class AuditTrailTests(APITestCase):
+
+    version_kwargs = {'version': '1'}
+    informatieobject_list_url = reverse_lazy('enkelvoudiginformatieobject-list', kwargs=version_kwargs)
+    objectinformatieobject_list_url = reverse_lazy('objectinformatieobject-list', kwargs=version_kwargs)
+    gebruiksrechten_list_url = reverse_lazy('gebruiksrechten-list', kwargs=version_kwargs)
+
+    def setUp(self):
+        super().setUp()
+
+        patcher_sync_create = patch('drc.sync.signals.sync_create')
+        self.mocked_sync_create = patcher_sync_create.start()
+        self.addCleanup(patcher_sync_create.stop)
+
+        patcher_sync_delete = patch('drc.sync.signals.sync_delete')
+        self.mocked_sync_delete = patcher_sync_delete.start()
+        self.addCleanup(patcher_sync_delete.stop)
+
+    def create_enkelvoudiginformatieobject(self):
```

Comment:
please mark helper methods as private by defining them with a leading underscore
```suggestion
    def _create_enkelvoudiginformatieobject(self):
```